DeepBlueRobotics/RobotCodeSimulator
# Webots gradle plugin

@alexander-mcdowell, if this works for you on a Mac and on a PC with a system-level install of Webots, please merge. If not, let me know what problems you are seeing. Thanks!

To test:

1. Switch to the webots-gradle-plugin branch
2. (Re)start the webots simulator manually (not via the bat file)
3. Run "Simulate Robot Code on Desktop" from VSCode (selecting the halsim_gui dll as usual)
4. Confirm you make the simulated robot run the autonomous routine by putting it is Autonomous mode with the wpilib sim gui
